rest/access-tokens/live-example/live-example.3.x.js

```diff
@@ -0,0 +1,36 @@
+const AccessToken = require('twilio').jwt.AccessToken;
+const PlaybackGrant = AccessToken.PlaybackGrant;
+
+// Used when generating any kind of tokens
+// To set up environmental variables, see http://twil.io/secure
+const twilioAccountSid = process.env.TWILIO_ACCOUNT_SID;
+const twilioApiKey = process.env.TWILIO_API_KEY;
+const twilioApiSecret = process.env.TWILIO_API_SECRET;
+
+// Create an access token which you will sign and return to the client,
+// containing the grant you will create in the next steps
+const token = new AccessToken(
+  twilioAccountSid,
+  twilioApiKey,
+  twilioApiSecret
+);
+
+// Create a PlaybackGrant resource for a specific PlayerStreamer
+// via the REST API
+const client = require('twilio')(twilioApiKey, twilioApiSecret, { 
+    accountSid: twilioAccountSid });
```

Comment:
Minor suggestion. I think the `client` should be initialized at the top of the example. since it is importing stuff. Even though in JS there is no rule against imports in the middle of the code, I think it is preferable to have all imports at the top, so that you can see all your dependencies at a glance.